addons/medical_engine/script_macros_config.hpp

```diff
@@ -7,44 +7,40 @@
         class My_AwesomeUnit_base;
         class My_AwesomeUnit: My_AwesomeUnit_base {
             class HitPoints {
-                ADD_ACE_HITPOINTS(2,2);
+                class HitHands;
+                class HitLegs;
+                ADD_ACE_HITPOINTS;
             };
         };
     };
 */
 
 // Our method for adding left and right arm and leg armor. Uses those selections
-// that are used for animations and therefore exist in all third party units
-// usage: arm_armor and leg_armor are the armor values of of HitHands and
-// HitLegs respectively.
+// that are used for animations and therefore exist in all third party units.
+// This used to take the armor values as parameters; it now inherits them from
+// the existing hitpoints instead.
```

Comment:
I think this would benefit from being more explicit to avoid regression:

```suggestion
// that are used for animations and therefore exist in all third party units.
// This used to take the armor values as parameters; it now inherits the values
// of `armor`, `passThrough` and `explosionShielding` from the existing hitpoints
// for vanilla consistency.
```

Reply:
@pterolatypus Does this rewording correctly reflect the reality and seem reasonable to you?